components/match2/wikis/rocketleague/match_legacy.lua

```diff
@@ -142,9 +142,14 @@ function MatchLegacy._convertParameters(match2)
 			and opponent2.score or 0
 		match.opponent2flag = player.flag
 	end
+
 	if match2.walkover then
 		match.resulttype = match2.walkover
-		match.walkover = nil
+		if match2.walkover == "ff" or match2.walkover == "dq" then
```

Comment:
shouldn't it be single quotes as per our standards?

Reply:
Rest of the file is in double quotes

Reply:
okay, maybe something we should clean up in the future ^^

Reply:
Oh yes